.github/workflows/precommit.yaml

```diff
@@ -2,11 +2,8 @@ name: Precommit tests
 
 on:
   push:
-    branches:
-      - '*'
+    branches: [main, develop]
   pull_request:
-    branches:
-      - '*'
 
 jobs:
   pre-commit:
```

Comment:
We need to remove the dependency on `install_dev_tools` or import that job too if it's necessary